### PR TITLE
feat(presets): Ignore .net 7 preview images

### PR DESCRIPTION
**Pull request by zharinov**
## Context

- Closes: #14477

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by viceice — DISMISSED** [review]
Still don't see much value🤷‍♂️

File: lib/config/presets/internal/workarounds.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ export const presets: Record<string, Preset> = {
       'workarounds:typesNodeVersioning',
       'workarounds:reduceRepologyServerLoad',
       'workarounds:doNotUpgradeFromAlpineStableToEdge',
+      'workarounds:ignoreDotnet7Preview',
```
Comment (zharinov) [inline lib/config/presets/internal/workarounds.ts:16]: Added to workarounds list, because I can't see how it can be useful otherwise

**Review by rarkins — APPROVED** [review]

**Comment by zharinov** [comment]
Let's merge this one or close the both

**Comment by viceice** [comment]
is on automerge, so just needed an update 😉

File: lib/config/presets/internal/workarounds.ts (hunk 1 above)
Comment (viceice) [reply lib/config/presets/internal/workarounds.ts:16]: that's what I want to avoid, as some users might want those updates?

**Review by viceice — CHANGES_REQUESTED** [review]
don't like to see this always enabled, as we only have one user who complain about that update

**Comment by zharinov** [comment]
Sorry, I thought it also requires one more reviewer. Trying to get rid of stale PRs, need to avoid producing newer ones

**Comment by viceice** [comment]
i think we should close those instead, otherwise we will forgot this and when v7 is released we'll get new bug reports about missing updates.

**Pull request closed without merging**